zeno_build/reporting/visualize.py

```diff
@@ -33,6 +38,16 @@ def visualize(
         raise ValueError("Length of data and labels must be equal.")
     if data_column not in df.columns:
         raise ValueError(f"Data column {data_column} not in DataFrame.")
+    if "ZENO_CLIENT_KEY" not in os.environ:
```

Comment:
nit: can we name this `ZENO_API_KEY`? It's named that in most places (e.g. where you copy it in the UI in Zeno Hub).